packages/plugins/src/plugins/eigenlayer/fetchers/deposit.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ethereumNativeAddress,
+  ethereumNetwork,
+  NetworkId,
+  PortfolioAsset,
+  PortfolioElement,
+  PortfolioElementType,
+} from '@sonarwatch/portfolio-core';
+import { getEvmClient } from '../../../utils/clients';
+import { Cache } from '../../../Cache';
+
+import { eigenPodManagerAddress, platformId } from '../constants';
+import { getAddress } from 'viem';
+import { abi } from '../abi';
+import tokenPriceToAssetToken from '../../../utils/misc/tokenPriceToAssetToken';
+import BigNumber from 'bignumber.js';
+
+/**
+ * Returns the deposit positions for a given owner
+ * @param owner - The address of the owner
+ * @param cache - The cache instance
+ * @returns The deposit positions
+ */
+export const getDepositPositions = async (owner: string, cache: Cache) => {
+  const client = getEvmClient(NetworkId.ethereum);
+  const podManagerAddress = getAddress(eigenPodManagerAddress);
+
+  const podUserAddress = await client.readContract({
+    address: podManagerAddress,
+    abi: [abi.ownerToPod],
+    functionName: abi.ownerToPod.name,
+    args: [getAddress(owner)],
```

Comment:
```suggestion
    args: [owner],
```
The caller needs to provide an `Address` param